_studio/shared/umc/codec/h264_dec/src/umc_h264_task_supplier.cpp

```diff
@@ -1090,7 +1090,25 @@ ViewItem & MVC_Extension::AllocateAndInitializeView(H264Slice * slice)
     }
     ViewItem * view = FindView(slice->GetSliceHeader()->nal_ext.mvc.view_id);
     if (view)
+    {
+        const UMC_H264_DECODER::H264SliceHeader *slice_hdr = slice->GetSliceHeader();
+
+        if (!slice_hdr->IdrPicFlag)
+            return *view;
+
+        H264SeqParamSet *pSps = const_cast<H264SeqParamSet*>(slice->m_pSeqParamSet);
+        int32_t dpbsize = CalculateDPBSize(m_level_idc ? m_level_idc : pSps->level_idc,
```

Comment:
Why this code block is here, but not in SetDPBSize()?

Reply:
SetDPBSize are called at different place and I am not sure it needs such change for all.